.github/workflows/tests.yml

```diff
@@ -95,6 +95,9 @@ jobs:
       - name: Install Rust
         uses: dtolnay/rust-toolchain@1.58.1
       - uses: Swatinem/rust-cache@v2
+        with:
+          # We want to avoid rebuilding the top-level crate each time.
+          cache-all-crates: "true"
```

Comment:
If this has a benefit I guess we'll want to propagate it to every Swatinem/rust-cache instance?